app/apollo/models/subscription.schema.js

```diff
@@ -72,6 +72,9 @@ const SubscriptionSchema = new mongoose.Schema({
     type: Date,
     default: Date.now,
   },
+  custom: {
+    type: String,
+  },
```

Comment:
Thinking we want this to be an array of key/value pairs. If I am reading correctly, this would be a single string